c/bpxskt.c

```diff
@@ -582,8 +582,10 @@ Socket *socketAccept(Socket *serverSocket, int *returnCode, int *reasonCode){
          returnCode,
          reasonCodePtr);
   if (returnValue == -1){
-    printf("Failed to accept new socket errno=%d reasonCode=0x%x\n",
-           *returnCode,*reasonCode);
+    if (socketTrace){
+      printf("Failed to accept new socket errno=%d reasonCode=0x%x\n",
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.